capstone/src/main/webapp/scripts/auth.js

```diff
@@ -33,22 +33,27 @@ function authenticateUser() {
  .then(response => response.json())
  .then((login) => {
     // need to set button's url to login url.
-    const loginUrl = login.url;
-    const loginButton = document.getElementById('login-btn');
-    loginButton.setAttribute('href', loginUrl);
+    const loginUrl = login.loginurl;
+    const loginButtons = document.getElementsByClassName("login-btn");
+    for(let i = 0; i < loginButtons.length; i++) {
+      loginButtons[i].setAttribute('href', loginUrl);
+    }
+    const loginStatus = login.loggedIn;
+
+    if (loginStatus) {
+      signUpUser();
+    }
  });
 }
 
 /*
- * Sign up the user. Placeholder code.
+ * Sign up the user for the app if they have not already signed up.
  */
 function signUpUser() {
-  // Get the user's first name and last name value through the form.
-
   // Send a POST request to the signup servlet with the user's name as params.
-  fetch('/signup')
-  .then(response => response.json())
-  .then((login) => {
-    // Do something with the response.
+  fetch('/signup', {method: 'POST'})
+  .then(() => {
```

Comment:
I'm wondering if a newly registered user should be sent to a place to update their name, or at least get a custom welcome message.

Reply:
Himani did call that out as part of the update. I think for now we can just add a TODO and add that for a follow-up PR

Reply:
Yes, I'll try to address this in my next PR!